File: src/iniciante/Bee1012.java
package iniciante;

import java.util.Scanner;

public class Bee1012 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        double A = input.nextDouble();
        double B = input.nextDouble();
        double C = input.nextDouble();
        double pi = 3.14159;

        double triangulo = (A*C)/2;
        double circulo = pi*(C*C);
        double trapezio = (A+B)*C/2;
        double quadrado = B*B;
        double retangulo = A*B;

        System.out.println(String.format(
                "TRIANGULO: %.3f"
                        +"\nCIRCULO: %.3f"
                        +"\nTRAPEZIO: %.3f"
                        +"\nQUADRADO: %.3f"
                        +"\nRETANGULO: %.3f",
                triangulo, circulo, trapezio, quadrado, retangulo
        ));
    }
}

File: beginner/bee1012.py
pi = 3.14159

def area(a, b, c):
    global pi
    print(f'TRIANGULO: {(a*c)/2:.3f}\n'
          f'CIRCULO: {pi * pow(c,2):.3f}\n'
          f'TRAPEZIO: {((a+b)*c)/2:.3f}\n'
          f'QUADRADO: {pow(b, 2):.3f}\n'
          f'RETANGULO: {(a*b):.3f}')

a, b, c = input().split()
a = float(a)
b = float(b)
c = float(c)
area(a, b, c)
